plugins/events-correlation-engine/src/main/java/org/opensearch/plugin/correlation/settings/EventsCorrelationSettings.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+package org.opensearch.plugin.correlation.settings;
+
+import org.opensearch.common.settings.Setting;
+import org.opensearch.common.unit.TimeValue;
+
+import java.util.concurrent.TimeUnit;
+
+import static org.opensearch.common.settings.Setting.Property.IndexScope;
+
+/**
+ * Settings for events-correlation-engine.
+ *
+ * @opensearch.api
```

Comment:
do you also want to mark as `@opensearch.experimental` for now? It's largely superficial but a nice flag if you think these Settings might change.

Reply:
marked as `@opensearch.experimental`